.orchestra/support/js-prerequisites/yarn

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
```

Comment:
Document what is going on here with the temporary directory.

Reply:
Done